src/PrestaShopBundle/Form/Admin/Type/EventListener/DisablingToggleListener.php

```diff
@@ -76,8 +76,6 @@ public function addDisablingToggle(FormEvent $event): void
         $newOptions = [
             'disabled' => $shouldBeDisabled,
             'attr' => [
-                'disabled' => $shouldBeDisabled,
-                'class' => $shouldBeDisabled ? 'disabled' : '',
```

Comment:
not sure how this will affect SpecificPricePriorities where this toggler should also be used. 
Need to double check once its merged (pr https://github.com/PrestaShop/PrestaShop/pull/26938)